src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -1740,6 +1740,32 @@ public void backticksQuotedAliasInJDBCResponseTest() {
         assertTrue(response.contains("\"alias\": \"name\""));
     }
 
+    @Test
+    public void caseWhenSwitchTest() throws IOException {
+        JSONObject response = executeQuery("SELECT CASE age " +
+                "WHEN '30' THEN '1' " +
+                "WHEN '40' THEN 2 " +
+                "ELSE '0' END AS cases FROM " + TEST_INDEX_ACCOUNT + " WHERE age IS NOT NULL");
```

Comment:
what is the expected result if age IS NULL?

Reply:
It says "all shards failed", which is the same with the behavior in functions when the "null" rows are not excluded in the functions/operators.